makefile

```diff
@@ -0,0 +1,19 @@
+OBJS=File.o Terminal.o Directory.o main.o
+CXXFLAGS=-O2 -std=c++11
+
+a.out: $(OBJS)
+	g++ -o a.out $(OBJS)
+
+main.o: main.cpp
+	g++ -c main.cpp $(CXXFLAGS)
+Terminal.o: Terminal.cpp
```

Comment:
gyanitom, hogy vannak headerek is, amiktol fugg :-)

Reply:
A makefile tökéletesen működik, úgy ahogy elvárt. Nem igazán értem a kérésed, hogy mit rakjunk rajta rendbe, főleg hogy teljesen ugyanilyen szerkezetű makefile-okat már jóváhagytál több csapatnál is, az előbb megnéztem. És Niki PR-ében a make && make clean azt csinálja hogy egy make-elés után letörli maga után a szemetet, azaz a .o fájlokat, a clean-nek nem pont ez a szerepe vagy ezt rosszul tudjuk?

Reply:
Napi 6 PR mellett sajnos becsuszik, hogy nem veszem eszre. Illetve ha mar az elso PR elott ott volt, akkor ugye nem is lattuk. De ha mar ennyire korbeneztel, azt is lathattad, hogy masoknal is szoltunk emiatt. Node: a makefile most azert mukodik jol, mert nincs ott semmi .o es igazabol ujrafordit mindent. Szoval a CI-ban nem fog problemat okozni, de ha aktivan lenne hasznalva, akkor igen. Szoval most az a baj, hogyha van egy x.h amit az x.cpp include-ol, es csak az valtozik, akkor nem fordul ujra a .o file, es ez linkelesi hibat fog okozni.

Reply:
make clean: azt tipikusan make elott szoktak futtatni, hogy clean build legyen. erre a CI-ban amugy nem sok szukseg van, mert ugye eleve nincs ott semmilyen korabbi o file. (amig nem hasznaljatok a cache-t arra, hogy az elozo build-bol eloszedjetek.)

Reply:
A Makefile-os történetet visszaállítottam anélkülire, az Niki commitjában majd rendbe lesz téve, eleve az az Ő commitjának célja.